ev3dev2/console.py

```diff
@@ -0,0 +1,104 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) 2015 Ralph Hempel <rhempel@hempeldesigngroup.com>
+# Copyright (c) 2015 Anton Vanhoucke <antonvh@gmail.com>
+# Copyright (c) 2015 Denis Demidov <dennis.demidov@gmail.com>
+# Copyright (c) 2015 Eric Pascual <eric@pobot.org>
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+# -----------------------------------------------------------------------------
+
+import os
+
+
+class Console():
+    """
+    A class that represents the EV3 LCD console, which implements ANSI codes
+    for cursor positioning, text color, and clearing the screen. Supports changing
+    the console font using standard system fonts.
+    """
+
+    def __init__(self, font="Lat15-TerminusBold24x12"):
+        """
+        Construct the Console instance, optionally with a font name specified.
+        This also resets the console: clearing it and turning off the cursor.
+        Parameter:
+
+        - `font` (string): Font name, as found in `/usr/share/consolefonts/`
+
+        """
+        self._font = None
+        self.set_font(font)
+
+    def text_at(self, text, column=1, row=1, clear_screen=False, inverse=False):

```

Comment:
`text_grid` for consistency?

Reply:
I too started with naming the function with `text_grid`, as found in the `Display()` class... but sensed simplicity in renaming it to `text_at`, as it didn't need to differentiate itself from `text_pixels`.